code/modules/clothing/head/misc.dm

```diff
@@ -196,31 +214,37 @@
 	icon_state = "bowler_hat"
 	desc = "For the gentleman of distinction."
 	body_parts_covered = 0
+	hidden_from_codex = FALSE
```

Comment:
why do you need to type "false" in every item if you can just do some funny things with abstract types so they're always hidden from codex??